bofire/strategies/predictives/botorch.py

```diff
@@ -267,6 +293,85 @@ def _postprocess_candidates(self, candidates: Tensor) -> pd.DataFrame:
         preds = self.predict(df_candidates)
         return pd.concat((df_candidates, preds), axis=1)
 
+    def _optimize_acqf_continuous(
+        self,
+        candidate_count: int,
+        acqfs: List[AcquisitionFunction],
+        bounds: Tensor,
+        ic_generator: Callable,
```

Comment:
Is `ic` inequality constraint? Maybe a more verbose variable name?

Reply:
No `ic_generator` means initial conditions generator, and that is the name for it that is also used in the botorch api. For this reason, I would like to keep it ;)